examples/jsm/renderers/nodes/core/VaryNode.js

```diff
@@ -0,0 +1,36 @@
+import Node from './Node.js';
+
+class VaryNode extends Node {
```

Comment:
Do you mind explaining the purpose of this class? Is it intended for managing varyings?

Reply:
Varying is used in `AttributeNode` but it send only the uniform object `vNormal = normal`. `VaryNode` can be util in case of send an varying with additional calcs, for example: `vNormal = normalize( normalMatrix * normal )`